### DISCUSSION TITLE
Zulip for Calyx

### DISCUSSION THREAD

**Discussion by rachitnigam** [cucapra/futil#1803] [General]
We've recently started getting a bunch of external people involved with Calyx but there is a bunch of friction is getting them help quickly. The two options we've been using are Github issues or directly adding folks to the lab Slack. As we move Calyx into it's own independent project, we should consider something like a Zulip instance to support new people. 

As a baseline, this also means moving some or all of our Calyx development conversation to Zulip so other people have visibility and we are consistently checking the instance to answer questions. This requires buy-in from the current Calyx-related projects which already have good enough workflows within the lab slack.

CC @EclecticGriffin @bcarlet @anshumanmohan @sampsyo who all have related projects

**Comment by sampsyo** [comment]
I definitely would like to explore it! As a proposal to consider, just to lock down the variables:

* Zulip is the software choice (over leading alternative contenders Slack, Discord, and Matrix).
* The scope should perhaps be "LATTE," with Calyx as one project within it?
* We can of course keep lots of research-related conversation in the Capra Slack, but the cost here is that we would want to try to put technical questions about Calyx-the-software in a different place. Which kinda means running two pieces of software.

**Reply by rachitnigam** [reply]
Summary from synchronous discussion: Rachit is in favor of the "maximalist" model of using Zulip. If there is buy-in from a project, then all implementation-side conversations about it will happen on Zulip. If we're writing a paper about the project, then that conversation can be moved to Slack.

Separately, we proposed:
- A open design meeting (ODM) in the style of the CIRCT project. We just put up the zoom room on the Calyx website and let anyone join. @sampsyo mentioned that this is particularly amenable to the "flavor of the week" model of the Calyx meetings anyways.
- A blog post describing the organizational changes to Calyx early in the year (moving to new repo, Zulip, ODM).
